scdl/scdl.py

```diff
@@ -643,6 +643,19 @@ def set_metadata(track, filename, album=None):
         artwork_url = user['avatar_url']
     artwork_url = artwork_url.replace('large', 't500x500')
     response = requests.get(artwork_url, stream=True)
+
+    #Write artwork cover to seperate file
+    if artwork_url and arguments['--write-cover']:  
+        invalid_chars = '\/:*?|<>"'
+        title = track['title']
+        title = title.encode('utf-8', 'ignore').decode('utf8')
+        title = ''.join(c for c in title if c not in invalid_chars)
+        
+        logger.info('Downloading track art...')
+        with open(title + '.jpg', 'wb') as img_file:
+            response.raw.decode_content = True
```

Comment:
I can't find anything about this in the documentation, can you explain why we need it ?

Reply:
As I understand, that is for the handling of the Content-Encoding header (compression, either gzip or deflate) in the response.